app.py

```
@@ -643,122 +1112,156 @@
+                           student_lookup_data=student_data, 
+                           course_attendance_details=course_attendance_details, 
+                           no_student_data=no_student_data, 
+                           overall_percentage=overall_percentage, 
+                           total_students=total_students,
+                           logged_in_username=session.get('username')) 
 
 
 if __name__ == '__main__':
-    app.run(debug=True, host='0.0.0.0', port=5000)
+
+    app.run(debug=True, host='0.0.0.0', port=5000)
```

Ruff (0.8.2)

Use of `debug=True` in Flask app detected

(S201)